app.py

```diff
@@ -1,4 +1,4 @@
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, session, redirect, url_for

```

Comment:
Forgot to delete this, will commit with it deleted after some people have checked the files out.